README.md

```diff
@@ -60,7 +60,9 @@ pub trait ResultSet {
 
```

Comment:
Thanks for this. You also need to update the `pub trait ResultSet` to add the `: Row` part